src/services/item/plugins/published/plugins/search/index.ts

```diff
@@ -17,12 +17,22 @@ export type SearchFields = {
 
 const plugin: FastifyPluginAsync = async (fastify) => {
   const searchService = fastify.search.service;
+  const actionService = fastify.actions.service;
 
   fastify.post(
     '/collections/search',
     { preHandler: fastify.attemptVerifyAuthentication, schema: search },
-    async ({ member, body }) => {
-      return searchService.search(member, buildRepositories(), body as MultiSearchParams);
+    async (request) => {
+      const { member, body } = request;
+      const repositories = buildRepositories();
+
+      // TODO: chnage type from graasp/sdk
+      const action = {
+        type: 'item-search',
+        extra: { ...(body as any) },
```

Comment:
Do you have an issue with the types here ? I am ok to keep it, but if possible I would prefer `unknown`.